charts/posit-chronicle/README.md

```diff
@@ -40,77 +40,104 @@ helm search repo rstudio/posit-chronicle -l
-```
+For additional information on deploying and configuring Chronicle agents,
+see the [Workbench](https://docs.posit.co/helm/charts/rstudio-workbench/README.html#chronicle-agent)
+or [Connect](https://docs.posit.co/helm/charts/rstudio-connect/README.html#chronicle-agent)
+chart documentation.
+
+## HTTPS Configuration
 
-And here is an example of Helm values for Connect, where a **Connect**
-API key from a Kubernetes Secret is used to unlock more detailed metrics:
+Chronicle can be configured to use HTTPS for secure communication. The
+`config.HTTPS` section of the configuration allows you to specify the certificate
+and key files to use for HTTPS. Both `config.HTTPS.Certificate` and
+`config.HTTPS.Key` are expected to be paths to files accessible by Chronicle.
+The `extraSecretMounts` value can be used to mount the certificate and key files
+into the Chronicle pod. Here is an example of how to do this, assuming that
+the certificate and key files are stored together in a Kubernetes TLS secret:
 
 ```yaml
-pod:
-  sidecar:
-    - name: chronicle-agent
-      image: ghcr.io/rstudio/chronicle-agent:2025.03.0
-      env:
-      - name: CHRONICLE_SERVER_ADDRESS
-        value: "http://chronicle-server.default"
-      - name: CONNECT_API_KEY
-        valueFrom:
-          secretKeyRef:
-            name: connect
-            key: apikey
+extraSecretMounts:
+  - name: chronicle-https
+    mountPath: /etc/chronicle/ssl
+    secretName: chronicle-https
+    items:
+      - key: tls.crt
+      - key: tls.key
+config:
+  HTTPS:
+    Enabled: true
+    Certificate: "/etc/chronicle/ssl/tls.crt"
+    Key: "/etc/chronicle/ssl/tls.key"
 ```
 
-Note that it is up to the user to provision this Kubernetes Secret for the
-Connect API key.
-
 ## Storage Configuration
 
-Chronicle can be configured to persist data to a local Kubernetes Volume, AWS S3, or both.
+Chronicle can be configured to persist data to local storage, AWS S3, or both.
+
+### Local Storage
 
-The default configuration uses a local volume, which is suitable if you'd like to
-access and analyze the data within your cluster:
+The default configuration will save data to a persistent volume, which
+is suitable if you'd like to access and analyze the data within your cluster.
+The below values show the default configuration for storage:
 
 ```yaml
+persistence:
+  enabled: true
+  accessModes:
+    - ReadWriteOnce
+  size: 10Gi
 config:
   LocalStorage:
     Enabled: true
-    Location: "/chronicle-data"
-    RetentionPeriod: "30d"
+    Location: "/opt/chronicle-data"
+```
+
+The `persistence` section configures the persistent volume claim in the
+cluster while the `config.LocalStorage` section directly applies to Chronicle's
+configuration file. The persistent volume will always mount to the path specified
+by `config.LocalStorage.Path` to avoid potential misconfiguration and data loss.
+
+By default, Chronicle requests 10Gi of storage. In most cases, this amount of
```

Comment:
I understand the rationale for enabling local storage as the default. I worry that some users will just deploy with this enabled and get themselves into trouble when they run out of space.

Could we add a `[!WARNING]` or something here? (https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

Reply:
I've added a callout to hopefully clarify this.